msgpack-jackson/src/main/java/org/msgpack/jackson/dataformat/MessagePackGenerator.java

```diff
@@ -38,7 +38,7 @@
         extends GeneratorBase
 {
     private static final Charset DEFAULT_CHARSET = Charset.forName("UTF-8");
-    private static ThreadLocal<MessagePacker> messagePackersHolder = new ThreadLocal<MessagePacker>();
```

Comment:
MessagePackGenerator is created each time when ObjectMapper#writeValue is called. So MessagePackGenerator stored MessagePacker in ThreadLocal to reduce the cost of instantiation.